requirements.txt

```diff
@@ -1,13 +1,11 @@
 #
-# This file is autogenerated by pip-compile with Python 3.8
+# This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
 #    pip-compile requirements.in
 #
 asgiref==3.5.2
     # via django
-backports-zoneinfo==0.2.1
-    # via django
```

Comment:
Did pip-compile remove these lines ?

Reply:
This file was not changed manually.

Instead, it was generated by pip-compile `6.12.2` with Python 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0] on linux:
```
pip-compile requirements.in
```